_data-prepper/common-use-cases/metrics-traces.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Deriving metrics from traces
+parent: Common use cases
+nav_order: 60
+---
+
+# Deriving metrics from traces
+
+You can use Data Prepper to derive metrics from OpenTelemetry traces. The following example pipeline receives incoming traces and extracts a metric called `durationInNanos`, aggregated over a tumbling window of 30 seconds. It then derives a histogram from the incoming traces.
+
+The pipeline contains the following multiple pipelines:
```

Comment:
Do we mean "The pipeline contains the following sub-pipelines"? "following multiple" doesn't work here.

Reply:
Per @dlvenable Data Prepper doesn't have sub-pipelines, so we can't use that terminology. Revised to read "The pipeline contains the following pipelines:"